### PR TITLE
Restructure the contact us page code 

### LINKED ISSUE #356: Add redirecting links to emails in contact us page
**Is your feature request related to a problem? Please describe.**
There are some email addresses on the Contact us page. When a user clicks on an email, it should redirect to the email app.

**Describe the solution you'd like**
Add anchor tags with mailto hrefs. Read this to get a basic idea: https://www.w3docs.com/snippets/html/how-to-create-mailto-links.html

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Are you working on it?**
No

### LINKED ISSUE #359: Restructure the contact us page code
**Is your feature request related to a problem? Please describe.**
We need to Remove those and restructure the code according to the material kit template and best practices. This will fix the bugs on mobile view also.

**Describe the solution you'd like**
Update the code structure. You can refer to the Starter template in https://demos.creative-tim.com/material-kit/docs/2.0/getting-started/introduction.html to get an idea about the correct code structure.

**Describe alternatives you've considered**
N/A

**Additional context**

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by VishwaAmarasinghe**
## Purpose

The purpose of this PR is to fix #356 , also fixes #359

## Goals
Make the heading correct, Add redirecting links to emails in contact us page , Fix mobile view issue in contact page , fix spellings mistakes ,  Restructure the contact us page code .

## Approach
I remove the text-roboto CSS class and also I remove it from index.html file . 
Next I add anchor tags to email with mailto: link to redirect it to email . Then I fix the mobile view issue in the contact page by adding bootstrap command .After that I remove unwanted classes, margins ,  padding .

### Screenshots
image

image
mobile view
  
### Preview Link

https://vishwaamarasinghe.github.io/sack-site/contact_us/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
#351

### CODE REVIEW

**Comment by VishwaAmarasinghe** [comment]
@jayasanka-sack  please give your idea.

**Comment by Piumal1999** [comment]
@VishwaAmarasinghe shall we use this pull request to complete https://github.com/acf-sack/sack-site/issues/359 as well?

**Comment by VishwaAmarasinghe** [comment]
> @VishwaAmarasinghe shall we use this pull request to complete #359 as well?

Ok brother

**Comment by VishwaAmarasinghe** [comment]
Is it okay now @Piumal1999 brother

File: contact_us/index.html [hunk 1]
```diff
@@ -3,40 +3,33 @@
...
   <!-- //Header -->
-
   <section class="container pb-5">
+    <h1 class="title">Contact us</h1>
     <div class="row">
-      <div class="col-sm-4 pb-3">
-        <h3 class="font-weight-bold pb-3">Primary Department</h3>
-        <p>Tel: <a href="tel:+94 081 221 3570">081 221 3570</a></p>
-        <p>E-mail: primary@sack.edu.lk</p>
+      <div class="col-sm-4 pb-3 pt-2">
```
Comment (Piumal1999) [inline contact_us/index.html:16]:
```suggestion
      <div class="col-sm-4 pb-3">
```
I think we don't need this padding here. Any special reason for adding that?

File: contact_us/index.html [hunk 2]
```diff
@@ -3,40 +3,33 @@
...
-<div class="main">
-  <section class="mt-5 container-fluid pt-5">
-    <div class="brand px-5 pt-5">
-      <div class="container">
-        <h1 class="title text-roboto">Contact us</h1>
-      </div>
-    </div>
-  </section>
+<div class="main py-5 pl-4">
   <!-- //Header -->
```
Comment (Piumal1999) [inline contact_us/index.html:12]: Remove this line
```suggestion

```

File: contact_us/index.html [hunk 3]
```diff
@@ -3,40 +3,33 @@
...
+        <p>Tel: <a href="tel:+94 081 221 3570" class="text-dark">081 221 3570</a></p>
+        <p><a href="mailto:primary@sack.edu.lk" class="text-dark">Email: primary@sack.edu.lk</a></p>
       </div>
       <div class="col-sm-4 pb-3">
-        <h3 class="font-weight-bold pb-3">Rev.Fr Principal</h3>
-        <p>Tel: <a href="tel:+94 081 221 3652">081 221 3652</a></p>
-        <p>E-mail: principal@sack.edu.lk</p>
+        <h3 class="font-weight-bold">Rev.Fr. Principal</h3>
+        <p>Tel: <a href="tel:+94 081 221 3652" class="text-dark">081 221 3652</a></p>
+        <p><a href="mailto:principal@sack.edu.lk " class="text-dark ">Email: principal@sack.edu.lk</a></p>
```
Comment (Piumal1999) [inline contact_us/index.html:24]:
```suggestion
        <p><a href="mailto:principal@sack.edu.lk" class="text-dark">Email: principal@sack.edu.lk</a></p>
```
Remove these extra spaces. Same in line 31

File: contact_us/index.html [hunk 4]
```diff
@@ -3,40 +3,33 @@
 title: Contact us | St. Anthony's College Kandy
 ---
 <!-- Header -->
-<div class="contact-us container-fluid mt-4">
-  <div class="row google-map">
```
Comment (Piumal1999) [inline contact_us/index.html:7]: I think we can also remove the custom CSS related to this google-map.

**Review by Piumal1999 — CHANGES_REQUESTED** [review]
Good job! @VishwaAmarasinghe 
I requested some minor changes. Please fix those and re-request the review.

File: contact_us/index.html (hunk 4 above)
Comment (VishwaAmarasinghe) [reply contact_us/index.html:7]: Ok brother

File: contact_us/index.html (hunk 1 above)
Comment (VishwaAmarasinghe) [reply contact_us/index.html:16]: Ok brother I will remove it. No special reason for adding this.

**Comment by VishwaAmarasinghe** [comment]
Is it okay now @Piumal1999 brother

**Review by Piumal1999 — APPROVED** [review]
Brilliant! @VishwaAmarasinghe 
Thank you for getting this done

**Pull request merged**